README.md

```diff
@@ -244,11 +244,10 @@ The result would be a very long document that is by itself not actually readable
 
 However, this allows us to start looking at TF-IDF from a class-based perspective.
 
-Then, instead of applying TF-IDF to the newly created long documents, we have to take into account that TF-IDF will take the number of classes instead of the number of documents since we merged documents. 
+Then, instead of applying TF-IDF to the newly created long documents, we replace inverse document frequency with an inverse class frequency to measure how much information a term provides to a class.
+
 All these changes to TF-IDF results in the following formula: 
```

Comment:
I think it would be good to add the paper that was used.

```suggestion
All these changes to TF-IDF were discussed in this [paper](https://arxiv.org/pdf/2203.05794.pdf) and result in the following formula: 
```